libnmstate/nm/applier.py

```diff
@@ -359,9 +364,19 @@ def _build_connection_profile(iface_desired_state, base_con_profile=None):
     if base_profile:
         con_setting.import_by_profile(base_con_profile)
     else:
+        iface_name = (
+            iface_desired_state[Interface.NAME][len(ovs.PORT_PROFILE_PREFIX):]
```

Comment:
I think this will collide with #514 . With a bond, the port may have more than one interface.
BTW, this is not an "ovs-interface", but any interface.

Can we just not set the the problematic property (None)?

Maybe @phoracek has an idea on how this can be resolved if the above option is not possible.

Reply:
Disclaimer: Not sure I fully understand what this function does.

The best solution would be to *not* set `interface.interface-name` on `ovs-port` at all, if we can leave it empty, that would be the best.

With OVS bond, there could be in theory two combinations:

- bond[bond1]=(eth1+eth2)
- bond[eth1]=(eth1+eth2) (does not make much sense, but should be valid in OVS DB)

The former should be just fine. The latter in theory too, if we used UUIDs everywhere. However, the same name would be used in two Interface states.

But again, not sure I fully get what are you doing here.